src/debug.ts

```diff
@@ -206,6 +206,8 @@ export class CXXConfigurationProvider extends ConfigurationProvider
         config.program = await this.expandAutoprojPaths((name) => ws.which(name), pkg.info, config.program)
         if (config.cwd) {
             config.cwd = await this.expandAutoprojPaths((name) => ws.which(name), pkg.info, config.cwd);
+        } else {
+            config.cwd = dirname(config.program);
```

Comment:
`program` might be a vscode variable expansion, and then cwd is broken (I left it out because it happened to me). I'd rather not set anything and add a "sane default" in the snippets we generate.

Reply:
Good point.. I will move this to ConfigManager

Reply:
Updated